hackathon_site/registration/models.py

```diff
@@ -99,21 +121,83 @@ class Application(models.Model):
         ],
         null=False,
     )
-    q1 = models.TextField(null=False, help_text="First question?", max_length=1000)
-    q2 = models.TextField(null=False, help_text="Second question?", max_length=1000)
-    q3 = models.TextField(null=False, help_text="Third question?", max_length=1000)
+
+    address1 = models.CharField(
+        max_length=255, null=True, blank=True, help_text="e.g. 35 St George St"
+    )
+    address2 = models.CharField(
+        max_length=255, null=True, blank=True, help_text="e.g. Apt. No. 13"
+    )
+    country = models.CharField(max_length=255, null=False, blank=False)
+    city = models.CharField(max_length=255, null=True, blank=True)
+    region = models.CharField(max_length=255, null=True, blank=True)
+    postal_code = models.CharField(max_length=6, null=True, blank=True)
+
+    how_many_hackathons = models.TextField(
+        null=False,
+        help_text="How many hackathons have you been to?",
+        choices=HACKATHON_NUMBER_CHOICES,
+        max_length=100,
+    )
+
+    what_hackathon_experience = models.TextField(
+        null=False,
+        help_text="If you’ve been to a hackathon, briefly tell us your experience. If not, describe what you expect to see and experience.",
+        max_length=1000,
+    )
+
+    why_participate = models.TextField(
+        null=False,
+        help_text="Why do you want to participate in NewHacks?",
+        max_length=1000,
+    )
+
+    what_technical_experience = models.TextField(
+        null=False,
+        help_text="What is your technical experience with software and hardware?",
+        max_length=1000,
+    )
+
+    hardware_preference = models.CharField(
+        null=False,
+        help_text="Please indicate your preference for using hardware in this hackathon",
+        max_length=10,
+    )
+
+    referral_source = models.TextField(
+        null=False, help_text="How did you hear about NewHacks?", max_length=1000
+    )
     conduct_agree = models.BooleanField(
-        help_text="I have read and agree to the code of conduct.",
+        help_text="I have read and agree to the "
+        '<a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH code of conduct</a>.',
```

Comment:
Can you add `rel="noopener noreferrer" target="_blank"` to all the `<a>` tags, otherwise clicking the link opens the site in the current window and when going back, you lose all the form inputs

Looks like there's only 4 `<a>`s